data_centre/data.py

```diff
@@ -78,8 +97,28 @@ def __init__(self, message_handler):
         self.midi_mappings = self._read_json(self.MIDI_MAPPING_JSON)
         self.analog_mappings = self._read_json(self.ANALOG_MAPPING_JSON)
 
-        
-
+        from utils import docs
+        docs.generate_mappings_doc("MIDI mappings", self.midi_mappings)
+        docs.generate_mappings_doc("OSC mappings", self.osc_mappings, column_one_header="OSC address")
+        docs.generate_mappings_doc("Key mappings", self.analog_mappings, column_one_header="Analogue input")
+        #quit()
+
+    def initialise_plugins(self):
+        #initialise plugin manager
+        self.plugins = plugin_collection.PluginCollection("plugins", self.message_handler, self)
+        self.plugins.apply_all_plugins_on_value(5)
```

Comment:
what is value 5 ?

Reply:
Ah nice spot- all that apply values on plugin stuff is leftover from the example that I pinched the plugins core from and can be removed